pom.xml

```diff
@@ -60,6 +60,10 @@
 			<artifactId>liquibase-core</artifactId>
 			<version>${liquibase.version}</version>
 		</dependency>
+		<dependency>
+			<groupId>org.hibernate.validator</groupId>
+			<artifactId>hibernate-validator</artifactId>
+		</dependency>
```

Comment:
Why this dependency is required?

Reply:
To use:
import jakarta.validation.Valid;

@Valid